pkg/kubelet/kubelet.go

```diff
@@ -1350,12 +1350,20 @@ func (kl *Kubelet) cleanupOrphanedPodDirs(pods []*api.Pod, runningPods []*kubeco
 		return err
 	}
 	errlist := []error{}
-	for i := range found {
-		if !active.Has(string(found[i])) {
-			glog.V(3).Infof("Orphaned pod %q found, removing", found[i])
-			if err := os.RemoveAll(kl.getPodDir(found[i])); err != nil {
-				errlist = append(errlist, err)
-			}
+	for _, uid := range found {
+		if active.Has(string(uid)) {
+			continue
+		}
+		if err := os.Remove(kl.getPodVolumesDir(uid)); err != nil {
```

Comment:
It's used here as a signal that would stop kubelet from removing the pod directory, regardless of what the error is. The other alternative is to read the directory and figure out whether it's empty or not. It doesn't seem any simpler.